bugsnag-android-ndk-shim/build.gradle

```diff
@@ -0,0 +1,28 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.4'
+    }
+}
+
+repositories {
+    google()
+    mavenCentral()
+    jcenter()
+}
+
+apply plugin: 'com.android.library'
+android {
+    compileSdkVersion 27
```

Comment:
It would have been nice to compile against 28 but I don't think should block the release - maybe something to address for the next release